assets/launcher/lang/sklmessages_pl_PL.properties

```diff
@@ -107,8 +107,8 @@ creator.field.gamedir=Używa domyślnego katalogu
 creator.modal.gamedir=Wybierz katalog gry
 
 # Versions List
-versions.category.vanilla=Vanilla
```

Comment:
Ja bym to nazwał "Gra bazowa" czy coś w tym stylu, bo Oryginał to jest nawet jeśli zainstalujemy mody